### PR TITLE
FIX: trialtime property

### PR DESCRIPTION
**Pull request by KatharineShapcott**
Removed unique as that made the _get_time function incorrect. 
Code is now faster and identical to .time property in continuous_data.

Author Guidelines
-----------------
- [x] Is the change set **< 400 lines**?
- [ ] Was the code checked for memory leaks/performance bottlenecks?
- [ ] Is the code running locally **and** on the ESI cluster?
- [ ] Is the code running on all supported platforms?

Reviewer Checklist
------------------
- [ ] Are testing routines present?
- [ ] Do **parallel** loops have a set length and correct termination conditions?
- [ ] Do objects in the global package namespace perform proper parsing of their input? 
- [ ] Do code-blocks provide novel functionality, i.e., no re-factoring using builtin/external packages possible?
- [ ] Code layout
  - [ ] Is the code PEP8 compliant?
  - [ ] Does the code adhere to agreed-upon naming conventions?
  - [ ] Are keywords clearly named and easy to understand?
  - [ ] No commented-out code?
- [ ] Are all docstrings complete and accurate?

### CODE REVIEW

**Comment by KatharineShapcott** [comment]
I think it's ready except that I'm not sure how/where to add tests for the new extra dimord columns in event data.

**Comment by pantaray** [comment]
Awesome! Thank you so much for your work @KatharineShapcott ! I'll try to get it asap.

**Comment by KatharineShapcott** [comment]
This test passes on my machine... I'm going to add back in something but it's unused so I don't see how this can make a difference...

File: syncopy/datatype/discrete_data.py [hunk 1]
```diff
@@ -617,6 +614,15 @@ def __init__(self,
...
         """
+        if dimord is not None:
+            # ensure that event data can have extra dimord columns
+            if len(dimord) != len(self._defaultDimord):
+                for col in self._defaultDimord:
+                    if col not in dimord:
+                        base = "dimensional label {}"
+                        lgl = base.format("'" + col + "'")
+                        raise SPYValueError(legal=lgl, varname="dimord")
+                self._defaultDimord = dimord
```
Comment (pantaray) [inline syncopy/datatype/discrete_data.py:625]: Nice! This is a really clever way to append columns to our required `sample` and `eventid` dimord.

File: syncopy/tests/test_selectdata.py [hunk 2]
```diff
@@ -657,71 +657,66 @@ def test_discrete_toitoilim(self):
...
+                                            discrete.trials[trlno][sel[trlno], :])
+                        if not isinstance(idx, slice) and len(idx) > 1:
+                            timeSteps = np.diff(idx)
+                            if timeSteps.min() == timeSteps.max() == 1:
+                                idx = slice(idx[0], idx[-1] + 1, 1)
+                        result.append(idx)
 
                     # check correct format of selector (list -> slice etc.)
-                    assert result == sel
+                    assert np.array_equal(result, sel)
```
Comment (pantaray) [inline syncopy/tests/test_selectdata.py:712]: Wow, this is super clean! Thanks a ton for updating the tests!

**Review by pantaray — APPROVED** [review]
This looks great, I think we can merge it right away. I'll include some `EventData` objects with custom `dimord` columns to also run the testing gamut just for the sake of completeness.  Thank you so much @KatharineShapcott for the great work!

**Pull request merged**